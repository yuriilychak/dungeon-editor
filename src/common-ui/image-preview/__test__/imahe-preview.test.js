import React from "react";
import { createShallow } from "@material-ui/core/test-utils";
import ImagePreview from "../image-preview";

describe("ImagePreview test",()=> {
    const preview = "test";

    it('default snapshot', () => {
        expect(createShallow()(
            <ImagePreview
                preview={preview}
            >
                <div>Test</div>
            </ImagePreview>
        ).html()).toMatchSnapshot();
    });

    it('snapshot with fixed width', () => {
        expect(createShallow()(
            <ImagePreview
                preview={preview}
                width={200}
            >
                <div>Test</div>
            </ImagePreview>
        ).html()).toMatchSnapshot();
    });

    it('snapshot with fixed height', () => {
        expect(createShallow()(
            <ImagePreview
                preview={preview}
                height={200}
            >
                <div>Test</div>
            </ImagePreview>
        ).html()).toMatchSnapshot();
    });
});
