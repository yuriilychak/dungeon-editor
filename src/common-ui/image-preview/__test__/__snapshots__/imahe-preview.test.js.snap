// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImagePreview test default snapshot 1`] = `"<div><span>Tooltip mock</span><div><div>Test</div></div></div>"`;

exports[`ImagePreview test snapshot with fixed height 1`] = `"<div><span>Tooltip mock</span><div><div>Test</div></div></div>"`;

exports[`ImagePreview test snapshot with fixed width 1`] = `"<div><span>Tooltip mock</span><div><div>Test</div></div></div>"`;
